workflow-beta2/tpl/deis-logger-fluentd-daemon.yaml

```diff
@@ -0,0 +1,40 @@
+#helm:generate helm tpl -d $HELM_GENERATE_DIR/tpl/generate_params.toml -o $HELM_GENERATE_DIR/manifest/deis-logger-fluentd-daemon.yaml $HELM_GENERATE_DIR/tpl/deis-logger-fluentd-daemon.yaml
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: deis-logger-fluentd
+  namespace: deis
+  labels:
+    heritage: deis
+spec:
```

Comment:
@jchauncey thanks. missing pieces added in https://github.com/deis/charts/pull/224/commits/ede8259117f7362f0828b79ba960732d05f612fb